releases/v1.7/schedule.md

```diff
@@ -0,0 +1,23 @@
+| 2025/08/27    | Kubernetes Release   | Kubernetes  | 1.34.0 release                                   |                                                                                                                                                                    |
+| **September** |                      |             |                                                  |
+| 2025/09/03    | KubeVirt Alpha 0     | KubeVirt    | Tag v1.7.0-alpha.0                               |                                                                                                                                                                    |
+| 2025/09/03    | KubeVirt Alpha 0     | KubeVirt    | Virtualization Enhancement Proposal (VEP) Freeze |                                                                                                                                                                    |
+| 2025/09/17    |                      | KubeVirt CI | CI lanes for provider are voting                 |                                                                                                                                                                    |
+| 2025/09/24    | KubeVirt Beta 0      | KubeVirt    | Tag v1.7.0-beta.0                                | Begin [release checklist](../release-checklist.md)                                                                                                                 |
+| 2025/09/24    | KubeVirt Beta 0      | KubeVirt CI | Mandatory pre-submit                             | Non-standard lanes (i.e. migrations, multus, ipv6) are to be bumped by stakeholder                                                                                 |
+| **October**   |                      |             |                                                  |                                                                                                                                                                    |
+| 2025/10/29    | KubeVirt Code Freeze | KubeVirt    | Branch release-1.7                               | Enter stabilization phase, review [list of quarantined tests](https://storage.googleapis.com/kubevirt-prow/reports/quarantined-tests/kubevirt/kubevirt/index.html) |
+| **November**  |                      |             |                                                  |                                                                                                                                                                    |
+| 2025/11/05    | KubeVirt RC 0        | KubeVirt    | Tag v1.7.0-rc.0 on release-1.7                   |                                                                                                                                                                    |
```

Comment:
I've wanted to ask this for a while now but why do we delay tagging `rc.0` after branching? Shouldn't we tag `rc.0` on `main` and then branch from that commit in one go?

Reply:
I believe we had some technical problem but we could try this in this release